src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -61,68 +59,64 @@
 
     private void validateOrganisationPolicy(String caseTypeId, CaseDataContent content) {
         final List<String> errorList = new ArrayList<>();
```

Comment:
Do we always execute ORGANISATION_POLICY_ROLE validations. There could be submit cases  where there is  ORGANISATION_POLICY_ROLE logic is not present .  It is easier just return it .

Reply:
we are checking in isOrgPolicyCaseAssignedRole(...)

Reply:
but it is always checking for ORGANISATION_POLICY when we could do it before executing all logic. DITO  for me is the same just and small comment